client/src/stories/assets/spriteSheet.js

```diff
@@ -0,0 +1,4 @@
xTT8JFF16AU1auROeMGcptlliHBwu2OLuWBUsiGs0tGE5nMToygr69O/Hbhx7Ez267Hdv27MVJixfCdwqQxo2BkVHs3Nf/Cx3hEB7b3rvvxvPPOBVdnZ0YHhtTlXXF3mdTXxeM6rb1qG5bxTRsJZgzuTwyuVwooNU+pY0bUd4NYNsxu8KEEEIIIScY9S6gv/PKi875P5effhJeeumlWLZiJVIFB0MjYxgdG8N4KqVvaaTTKaQyGaT0tlRqXI3kzhZsxBsalWhubYjjV/f/FvevehzLFi1AT1cHNu/YhT37B74GYLv+zN3ZXP7Amhc2vnbpgnlYMGc2coUCMtlsOI0wGJ7ihCJaRLJyvjM5pDJZpLI5ZPJ55G2dd650nQ3fBPC9Y3BdCSGEEEJOWOpaQC+e2/Pt73/5c52ts3qw78AgUqk0HJUllkWDeeSVwyv3weOCXVALC2UfJxS7jlpcOJ7OqgWEpy1fgv4D+/HbJ9Zg+eIFGEunsXPv/n8vc4GfTGWyjz68+pkLCrY9e3bXDHR1tKuFgOrm2MgVbGRzeaSzOXXssVRa3Yt4FjFtquv86sIZWjh/7GhdS0IIIYSQeqGeBfRL3vjqV3ziwnPPwa49vWrkte3aqnFD8sUFFY2wVWzC0ZMDXR2xMDVxrghpLXplkmBBxysWz5sr/dJ4dtM21awxODr2DbuZVgAABqVJREFUUwBbyz5fnv/ztt29T63duOVSx3VniBjO5rIYTaUxNp5Sonl0LIWxdEaJ53ROxHMg3l3fQxXlLBNh5LP+BMAPjtqVJIQQQgipI2qP1Dvx+d0P/v4Ll0uE4sDgsBLGxmUOnGBbPRZRHEQp7NAdLr4uQrugFvGJgA5ed9RCv6aGJNZt24V9AyNyIa8GcN8EV1RGfd8Us/CGlqam5bGYtdTS9R2mzzkY6a06ncd0JV1au9qjWoyvAfCgzGvh/wcmhBBCCJk+6rWF44qrX3rp5aeffBK27dqtnGXjNKubW8wfu9GFfZEFfmYfNVbbc8N6OYkjy+PxTBZL5vYgkytgNJVxJzkfKaK+2fNxs0Q0ACwBMBPAPAAX6X2e10J5uxbQkx2TEEIIIYRMA/UooGNdnR0//8A734bh0dGw+UIEtEQ4XL2ATz3WojmIb5gBJU7JYj81XtvzgwmBuo5OTQp0XJWJntnRJgL6YKMyO/VNXOW7p+k6EEIIIYSQQ6AeR3n/71995KaeOd0zMTA0rPqVJYqhquAi9XGObTLPgVi2jfPsuOFNXlfTA0PxbHqcpU4O4f6yXvHYf21CCCGEEHIkqDcB/X+uvvyS17/sxRdgR+8+tQZPdS3bgfMsjrOa5heJbpS60MWuZnGig3HbOroRThP0QjFtupkBLDz2X50QQgghhBwJ6kpAd3V2fO8j73kH+geGQpFcUNnnSCWdim44RfHsRHPPRXHtRsWyiGff3AfuszRqyOLCeDwmkwyvA9B8HFwCQgghhBBymNSTgP70Tde9ZV5PdzcGh0dU9EIq6P7/9u5fx60iiuP48a5sx94A4t8mGysIhEgqRIdQCgok6Oh4ChoaCp6ChkegpKFAdIECCUERGhBSSgqQFtBuAiSxPXPmopk5M/feXTpbctj7/UiWdze2tTtpfjo6c05vPJ3Lfc/pQqDP2/+8tn3P9bkTntVWbwe1EJ3mMjfpNbHtIwbp6WR8Q0QWj8EZAAAAYENDCdCXFlcPP3zr1uvy6/GxxAlxGmK12deAXCrMNUynlo7OhUGrRHerzmqXBtVaOULZCNjkf0sruC1E22QNAAAA/M8NJUC/9OrNV54/mM/S+u0Ubn2wvue8GCUtQ+mMqlNtWzbSc9B+j3Np3SiXBkOuPIcUnpta2XYWukXktd0fAwAAADY1lACt14+uykhGFnw1Td/IATcvR6ktGr6dvKGdCnUKylZZrp8RH6niHFJ4bnrV52AXDrVcJPxn98cAAACATQ0lQL85nU5qj3LqXQ7tKDr15yds9Oc+hxSMg6q9N6Qqcx5Xl5enpCAdJLd2aO6NjlVun96bfofndn4KAAAA2NhQAnSaw5wqxBaAa3vGuW2D2pnKoXZJsPmP9g1bmhLsc23yRnytr2E8Xzi0Hui3d34KAAAA2NhQAvQDtUpwsEpyb6pGZ7Ngd9tgXpQS32frujstHKlto5HatpHCc5r9XKrZIc+BbgP0EJfWAAAAXDiDCXV7o1EKwb0KtD2c971xdblyrHWuc7oUGNs1LFDnn1t4PtP7HIO6s/7nPPKukdzBIbrrMwAAAMDmBhOg57NpbscoPdCl/7lbjS5bB32w6RrWgnFm8kaKxKXyXJentKPrtATo0IZrKtAAAAAXw1BC3ehgPm+rwvURzvU/a6k+14BsM57VLg522jW6z2V0nbfPLL3TFp6j6W6PAAAAANswlAD99HQ8ljgH2tVV3PboBOo0ck7LkhRtLwqqhegUhsuFQcltHPZ9ODO67kz/c/TNbo8AAAAA2zCUAP3lp59/ISen9+SFa0d5UkZ3dJ1Vn3tznevkjfwcv2/Dc6f3ObRru9e2lKVO72jDc/Tx7v58AAAAbMv+QE7ylz9OTu/e+fHn9xZXDuXo8BkJ6uWvvx/IcrUS51yv11nriDrrl9Z2UYr2LiHmqnOsaq/WTlbOp6/Xztv85xqgPxCR27s9AgAAAGzDUAJ09NPJvfuf3f72+xd///P0xqXpRJ68PJf9vT3x3rWX/2qLR6itHaETmusFwbIsJWgKzMu1k+VqnZ4767ujj6g+AwAAXByjgf5fxqUm7798ffHO4sqz86cO5jKdjFOYDiEvWIkhOK74dq70Src9zqnK7C04r9bycLmSh6u1OOd9aJpjG1n3lYh8IiI/PAZ/LwAAALZkqAG6OBSRWyLy7hMHszcuz2bXJuP9/dh5kfqZnX8UQnjU2BbDKIS2jUNVl17DfRG5IyJfi8h3IvJbfFlqmAYAAAAAAACAwRKRfwEIeCt3JN4mJgAAAABJRU5ErkJggg==";
```

Comment:
Any reason you saved it as png URI string?
I guess it's more common to save the sprite in a .png file. It makes it easier to maintain.

Reply:
We did it this way in the [storybook-corona-game](https://github.com/lmachens/storybook-game/blob/master/src/assets/images.js) as well. Yesterday I had problems with the png. The src was always undefined. Same occurred in the storybook-corona-game. It would definitely be easier if I could use the png. Couldn't solve the problem and was happy to see the image somehow 🙈

Reply:
Fair enough. Let's take a look at this issue tomorrow (please remind me)

Reply:
I figured it out myself :D The picture was too big. Now it works 😄